src/game_renderer/ViewFrustumCulling.cpp

```diff
@@ -0,0 +1,116 @@
+#include "ViewFrustumCulling.h"
+
+namespace ViewFrustumCulling {
+
+   // Code from CPE-476 lab
+   glm::vec4 * GetViewFrustumPlanes(glm::mat4 P, glm::mat4 V) {
+     glm::vec4 Left, Right, Bottom, Top, Near, Far;
+     glm::vec4 *planes = (glm::vec4 *)malloc(sizeof(glm::vec4) * 6);
```

Comment:
Oh, I just realized that this is a list of glm::vec4s. How about using a std::vector to store them, and return that instead of the pointer?